### ISSUE TITLE
Dont work "GitHub issues by-label" with npm scoped packages

### ISSUE THREAD

**Issue by IgorNovozhilov** [badges/shields#1870]
Dont work "GitHub issues by-label" with npm scoped packages.
My issues label equal to the name of the module.

my shield dont work: https://img.shields.io/github/issues-raw/IgorNovozhilov/ndk/@ndk/cfg.svg
in github: https://github.com/IgorNovozhilov/ndk/labels/@ndk/cfg

shields: https://shields.io/#/examples/issue-tracking
shield: **GitHub issues by-label**

**Comment by paulmelnikow** [comment]
Hey, thanks for your question.

I think what you want is this:
https://img.shields.io/github/issues-raw/IgorNovozhilov/ndk/cfg.svg

![](https://img.shields.io/github/issues-raw/IgorNovozhilov/ndk/cfg.svg)

Hope that helps!

**Comment by IgorNovozhilov** [comment]
Yes, I have considered this decision earlier. But I wanted to use full module names as a name of label. Moreover, github allows you to do it.

**Comment by paulmelnikow** [comment]
Ahh, I gotcha. Your label has an @ and a slash in it: **@ndk/cfg**. Yea, that’s something we could fix.

**Comment by IgorNovozhilov** [comment]
the best, it's working ![](https://img.shields.io/github/issues-raw/IgorNovozhilov/ndk/@ndk/cfg.svg)
